### PR TITLE
Stop installing Faraday

### PR DESCRIPTION
**Pull request by HarlemSquirrel**
Using Net::HTTP from standard-lib should save some deploy time.

Also added to helper.rb to DRY things up some.

### CODE REVIEW

**Comment by dalexj** [comment]
said this in slack, but would rather this go toward removing faraday

originally had included it to perform GET requests with headers easily which is no longer occurring

File: lib/helper.rb [hunk 1]
```diff
@@ -9,12 +9,15 @@ def env_fetch(key)
...
+  req = Net::HTTP::Post.new(uri)
+  req["Content-Type"] = "application/json; charset=utf-8"
+  req["Authorization"] = "Bearer #{slack_api_token}"
+  req.set_form_data(channel: slack_channel_id, blocks: blocks)
+
+  response = Net::HTTP.start(uri.hostname, uri.port) do |http|
+    http.request(req)
+  end
+
+  response.value # Raises an error for non-200 response
```
Comment (dalexj) [inline lib/helper.rb:22]: dont think we really want this error. fine to just swallow a non-200 and move on

File: lib/helper.rb [hunk 2]
```diff
@@ -8,13 +9,14 @@ def env_fetch(key)
...
+def slack_message(blocks, slack_channel_id: env_fetch("SLACK_CHANNEL_ID"), slack_api_token: env_fetch("SLACK_API_TOKEN"))
+  uri = URI("https://slack.com/api/chat.postMessage")
+  req = Net::HTTP::Post.new(uri)
+  req["Content-Type"] = "application/json; charset=utf-8"
+  req["Authorization"] = "Bearer #{slack_api_token}"
+  req.set_form_data(channel: slack_channel_id, blocks: blocks)
+
+  Net::HTTP.start(uri.hostname, uri.port) do |http|
+    http.request(req)
+  end
```
Comment (dalexj) [inline lib/helper.rb:21]: is this any different from the shorthand with `Net::HTTP.post`? would like to use the shorthand
```ruby
Net::HTTP.post(
  URI("https://slack.com/api/chat.postMessage"),
  { channel: slack_channel_id, blocks: blocks }.to_json,
  {
    "Content-Type" => "application/json; charset=utf-8",
    "Authorization" => "Bearer #{slack_api_token}"
  }
)
```

Comment (HarlemSquirrel) [reply lib/helper.rb]: Updated

**Pull request merged**